File: Prechart.Portal/src/app/modules/werkgever/models/werkgever.ts
import { Klant } from "./klant";
import { WhkPremies } from "./whkPremies";

export class Werkgever {
  public id: string;
  public klant: Klant;
  public naam: string;
  public sector:number;
  public fiscaalNummer: string;
  public loonheffingenExtentie: string;
  public omzetbelastingExtentie: string;
  public whkPremies: WhkPremies[];
  public datumActiefVanaf: any;
  public datumActiefTot: any;
  public dateCreated: any;
  public dateLastModified: any;
  public actief: boolean;
}

File: Prechart.Service.Werkgever/Database/Models/Werkgever.cs
namespace Prechart.Service.Werkgever.Database.Models;

public class Werkgever
{
    public int Id { get; set; }
    public string WerkgeverMongoId { get; set; }
    public string KlantMongoId { get; set; }
    public string Naam { get; set; }
    public int Sector { get; set; }
    public string FiscaalNummer {get;set; }
    public string LoonheffingenExtentie { get; set; }
    public string OmzetbelastingExtentie { get; set; }
    public DateTime DatumActiefVanaf { get; set; }
    public DateTime DatumActiefTot { get; set; }
    public bool Actief { get; set; }
}